internal/chart/application_chart.go

```diff
@@ -123,6 +127,13 @@ func New(application *ketchv1.App, framework *ketchv1.Framework, opts ...Option)
 		return nil, err
 	}
 
+	appType := "Deployment"
```

Comment:
it'll be possible to use `Type: app.Spec.GetType()` there

Reply:
Changed